api/core/tools/tool_manager.py

```diff
@@ -236,7 +236,12 @@ def _init_runtime_parameter(cls, parameter_rule: ToolParameter, parameters: dict
 
     @classmethod
     def get_agent_tool_runtime(
-        cls, tenant_id: str, app_id: str, agent_tool: AgentToolEntity, invoke_from: InvokeFrom = InvokeFrom.DEBUGGER
+        cls,
+        tenant_id: str,
+        app_id: str,
+        agent_tool: AgentToolEntity,
+        variables_inputs: Optional[Mapping[str, Any]] = None,
+        invoke_from: InvokeFrom = InvokeFrom.DEBUGGER,
```

Comment:
I'm a bit skeptical of defining `variables_inputs` here, because this method is used in contexts where `variables_inputs` isn't available.